Classes/Bookmark/BookmarkStore.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  BookmarkStore.swift
+//  Freetime
+//
+//  Created by Rizwan on 18/10/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+final class BookmarksStore {
+    static let shared = BookmarksStore()
+    
+    private let archivePath: String = {
+        return FileManager.default
+            .urls(for: .documentDirectory, in: .userDomainMask)
+            // crash intentionally on startup if doesn't exist
+            .first!
+            .appendingPathComponent("bookmarks")
+            .path
+    }()
+    private var _bookmarks: Set<BookmarkModel>
```

Comment:
Why the `_`?

Reply:
Inspired from `SearchRecentStore`. Just to differentiate between on memory bookmarks.